docs/source/algorithms/LoadNexusLogs-v1.rst

```diff
@@ -65,10 +65,12 @@ Items missing from the Nexus file are simply not loaded.
 
 If the nexus file has a ``"proton_log"`` group, then this algorithm will do some event filtering to allow SANS2D files to load.
 
-The ``AllowList`` option skips loading of the entire sample log, and loads only log entries specified in the list. This can be
-useful if only particular log entries are needed. Likewise, the ``BlockList`` option will skip loading log entries specified in
-the list, while loading all other sample log entries that are not found in the list. For both of these options, log entry names
-should be separated by a space. Both ``AllowList`` and ``BlockList`` cannot be set at the same time.
+The ``AllowList`` option skips loading of the entire sample log, and loads only log entries specified in the list.
+This can be useful if only particular log entries are needed.
+Likewise, the ``BlockList`` option will skip loading log entries specified in the list (regular expression is supported
```

Comment:
This would benefit from a link which describes the allowed regexp

Reply:
I will update the doc to include the additional information.